drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -636,26 +637,156 @@ static int bq274xx_gauge_configure(const struct device *dev)
 		return -EIO;
 	}
 
+#ifdef CONFIG_PM_DEVICE
+	bq274xx->pm_state = DEVICE_PM_ACTIVE_STATE;
+#endif
+
+	return 0;
+}
+
+#ifdef CONFIG_PM_DEVICE
+static int bq274xx_enter_shutdown_mode(struct bq274xx_data *data)
+{
+	int status = 0;
+
+	status = bq274xx_control_reg_write(data, BQ274XX_UNSEAL_KEY);
+	if (status < 0) {
+		LOG_ERR("Unable to unseal the battery");
+		return -EIO;
+	}
+
+	status = bq274xx_control_reg_write(data, BQ274XX_UNSEAL_KEY);
```

Comment:
Is it correct to write `BQ274XX_UNSEAL_KEY` twice?

Reply:
Yes, that is all according to the bq27421 technical reference. You can also see the same thing inside `bq274xx_gauge_configure` function.